README.md

```diff
@@ -2,14 +2,39 @@
 ## Prerequisites
 - Java 21
 - Docker
+## Used technologies
```

Comment:
i'd say "Technology Stack". also merge this one with `# Prerequisites` and add a note in which you indicate that Docker is mandatory.